packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -1,20 +1,25 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { overrideArgs } = storybookHelpers('sd-dialog');
 const { argTypes, parameters } = storybookDefaults('sd-dialog');
 const { generateTemplate } = storybookTemplate('sd-dialog');
 
 /**
- * **Disclaimer** : Input-based controls for `sd-dialog` are unavailable on this Docs page due to a [Storybook bug](https://github.com/solid-design-system/solid/issues/702). Please use the individual feature stories to access these controls.
+ *
+ * Appears over other content. It requires an interaction from the user before they can return to whatever is underneath.
+ *
+ *  **Related templates**:
+ * - [Dialog with scrollable content](?path=/docs/templates-dialog-with-scrollable-content--docs)
+ *
  */
 
```

Comment:
- Use the `headline` slot to add a headline.
- Use the `default` slot to add main content.  Default slot is always scrollable.
- Use the `footer` slot to add action elements. Footer slot is always fixed.

Reply:
Updated